lib/cldr-plurals/javascript_emitter.rb

```diff
@@ -84,6 +82,11 @@ def emit_value_set(value_set, operand, operator)
         end
       end
 
+      def emit_range_check(range, operand)
+        # a..b represents all *integers* between a and b, inclusive.
+        (range.start..range.finish).to_a.map { |v| "(#{emit(operand)} == #{v})" }.join(" || ")
```

Comment:
you end up with `(n == 11) || (n == 12) || ... || (n == 99)`... that's a lot of comparisons if n is 99.